CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 0.60.0      2024-10-15
+
+* Update `get_column_stats` for Postgres 17
+  - The function signature has changed, so must be dropped and recreated
+    https://pganalyze.com/docs/install/troubleshooting/column_stats_helper
+* Handle ambiguous log lines more reliably
+  - This fixes parsing of some log lines for Google CloudSQL
+* Include log text in snapshots instead of uploading to S3
```

Comment:
We should reword this to make it clear that its changing the way these uploads are done (i.e. the old way is removed), and put an explicit notice of the impact of this change for Enterprise installs running the collector separately.

Reply:
Yeah, maybe something like
```suggestion
* Change log upload to include log text directly in snapshots, instead of uploading it separately to S3
```
I think this with the warning I suggested above makes the situation clear.